### PR TITLE
task(deps): Upgrade ember-test-waiters to ^2.0.0

### PR DESCRIPTION
**Pull request by scalvert**
Relates to https://github.com/NullVoxPopuli/emberclear/pull/558

### CODE REVIEW

**Comment by scalvert** [comment]
cc @chriskrycho

**Comment by NullVoxPopuli** [comment]
Looks like ember-router-scroll will need to update, too. (Just trying to trace where this issue came from in my app from my phone)

File: addon/scheduler.ts [hunk 1]
```diff
@@ -70,9 +70,7 @@ export function reset(): void {
   _whenRouteDidChange = _defer(APP_SCHEDULER_LABEL);
   _whenRouteIdle = _whenRouteDidChange.promise.then();
 
-  if (waiter instanceof TestWaiter) {
-    waiter.items.clear();
-  }
+  waiter.reset();
```
Comment (chriskrycho) [inline addon/scheduler.ts]: 😍

**Comment by snewcomer** [comment]
ember-router-scroll updated.  Thank you all!

**Pull request merged**